hack/jenkins/cron/cleanup_and_reboot_Linux.sh

```diff
@@ -36,11 +36,83 @@ logger "cleanup_and_reboot is happening!"
 # kill jenkins to avoid an incoming request
 killall java
 
+# clean minikube left overs
+echo -e "\ncleanup minikube..."
+for user in $(lslogins --user-accs --noheadings --output=USER); do
```

Comment:
We can assume only jenkins and root (for the none driver) as users here here instead of using the for loop across all users

Reply:
ok, i've hardcoded only jenkins and root user as targets for the cleanup